pelican/plugins/test/data/pelicanconf.py

```diff
@@ -1,4 +1,5 @@
 import pathlib
+from pelican import settings
```

Comment:
Should probably add a blank line in between these two lines.

Reply:
I'm not sure why it had disappeared